### DISCUSSION TITLE
Node avoiding router

### DISCUSSION THREAD

**Discussion by wernerkeilholz** [vuetifyjs/vuetify#16575] [Q&A] [answered]
I looked at draw2d.layout.connection.ManhattanConnectionRouter and this looks pretty much like what I'm trying to do.

However, in my use case we'd like the links to avoid existing nodes (automatically add manhatten-style bends in such a way that no line ever crosses a node). 

Does that exist ?

**Comment by nekosaur** [comment, marked as answer]
I think you posted this in the wrong repository

**Comment by wernerkeilholz** [comment]
You are right, sorry for that.
This was about the Draw2D JavaScript graph and diagramming lib, no idea how it got mxed up.

**Reply by wernerkeilholz** [reply]
Actually, when I start at 
https://freegroup.github.io/draw2d/#/examples/section1
then click 'Latest Release', then 'Discussion'... I get here

**Reply by nekosaur** [reply]
Interesting :) But that's something for them to fix.
